static/robots.txt

```diff
@@ -1,4 +1,5 @@
 User-agent: *
 Sitemap: https://docs.apify.com/sitemap.xml
 Disallow: https://docs.apify.com/api/v2-old
-Disallow: https://docs.apify.com/api/v2-new
+Disallow: https://docs.apify.com/api/v2-redoc
+Disallow: https://docs.apify.com/api/v2
```

Comment:
We want the `/api/v2` to be indexed, and this would disallow that, wouldn't  it?

```suggestion
```